sep_021.md

```diff
@@ -94,7 +95,13 @@ Examples <a name='examples'></a>
 
 ### 3.1 Flow Cytometry <a name="example1"></a>
 
-**Figure 3:** An example of using the classes from this 
+**Figure 3:** An example of using the classes from this
+
+### 3.2 Interlab Example <a name="example2"></a>
+
+![ExperimentalData class UML diagram](images/sep_021_interlab_example.png "ExperimentalData class UML diagram")
```

Comment:
This is still not quite there: there should be one ExperimentData (and associated Attachments) per Implementation --- the point of the Experiment is to be able to cluster together the different sample/data pairs, and the point of the ExperimentData is to be able to batch or dissect Attachments.

Reply:
One last tweak: could you please separate the "positive control" and "study construct" "wasGeneratedBy" lines? They are currently overlain, such that it looks like a single multi-head/multi-tail arrow.  As soon as that is done, I will approve the pull request.

Reply:
Done!

Reply:
Is this actually pushed? I don't see anything different in the pull request right now.

Reply:
My bad, I had forgotten to export the image after changing the UML. Should be done now.